examples/language/lib/class_modifiers/ex5/sealed.dart

```diff
@@ -7,16 +7,21 @@ class Truck implements Vehicle {}
 
 class Bicycle extends Vehicle {}
 
-// ERROR: Can't be instantiated.
+// ERROR: `Vehicle` can't be instantiated because
+// it is marked `sealed` and therefore, implicitly abstract.
 Vehicle myVehicle = Vehicle();
 
-// Subclasses can be instantiated.
+// Subclasses of a sealed class can be instantiated unless also restricted.
 Vehicle myCar = Car();
 
-String getVehicleSound(Vehicle vehicle) {
-  // ERROR: The switch is missing the Bicycle subtype or a default case.
-  return switch (vehicle) {
-    Car() => 'vroom',
-    Truck() => 'VROOOOMM',
-  };
+extension VehicleSounds on Vehicle {
+  String get sound {
+    // ERROR: The switch does not exhaustively account for
+    // all possible instances of a `Vehicle`.
```

Comment:
It is always tricky to describe the situation where an object `o` has run-time type `B`, and `A` is some supertype of `B`. We don't usually say that `o` "is an instance of `A`" because that tends to mean that it has run-time type `A`. I often say that "`o` has type `A`" or "`o` is typable as an `A`". I'm then relying on the ability to claim that this implies that `A myA = o as dynamic;` will succeed, and hence `A` is one of the types that `o` "has". This might suffice:

```suggestion
    // all possible objects of type `Vehicle`.
```